graphql-schemas/schema.yml

```diff
@@ -3536,7 +3536,8 @@ confs:
   interface: GlitchtipProjectAlertRecipient_v1
   fields:
   - { name: provider, type: string, isRequired: true, isContextUnique: true }
-  - { name: url, type: string, isRequired: true, isContextUnique: true }
+  - { name: url, type: string, isContextUnique: true }
+  - { name: urlSecret, type: VaultSecret_v1, isContextUnique: true }
```

Comment:
Can `isContextUnique` support `object`?

Reply:
TBH, I don't know. Currently, the `isContextUnique` isn't working at all, but @geoberle is already on it

Reply:
i'm going to open a PR for q-v soon that will adress the issues with context-unique.
`isContextUniqe` also supports objects.